cabal-testsuite/PackageTests/AutogenModules/Package/setup.cabal.out

```diff
@@ -1,6 +1,7 @@
 # Setup configure
 Configuring AutogenModules-0.1...
-Error: cabal: An 'autogen-module' is neither on 'exposed-modules' or 'other-modules'.
```

Comment:
I don't see this in the patch, so I presume it's catching up from an earlier batch… one should use "nor" with "neither", not "or".

Reply:
yes this is how it was earlier, but I can change this as part of this batch? Thanks

Reply:
If it used to be this way then maybe we want some discussion, or maybe you can point me at an earlier discussion?

Reply:
i think its fine to change it to nor, and we may as well do it here. i think suganya just meant they were trying to change only what we had specced out, and not make language changes as well.

Reply:
Thanks for adding your comments @gbaz .
@geekosaur Sorry if I wasn't clear before, As Gershom mentioned I was not sure if I can make any changes outside of assigned and agreed specs related to error codes.
But I see your point and its valid, I can make this change as part of this PR.
Thanks.